library/include/G4CMPConfigMessenger.hh

```diff
@@ -99,6 +103,16 @@ private:
   G4UIcmdWithABool*   ehCloudCmd;
   G4UIcmdWithABool*   recordMinECmd;
 
+  // Empirical Lindhard Model Macro Commands
+  G4UIcmdWithABool* EmpiricalEnergyDependentKCmd;
+  G4UIcmdWithADouble* EmpiricalkFixedCmd;
+  G4UIcmdWithADouble* EmpiricalklowCmd;
+  G4UIcmdWithADouble* EmpiricalkhighCmd;
+  G4UIcmdWithADoubleAndUnit* EmpiricalElowCmd;
+  G4UIcmdWithADoubleAndUnit* EmpiricalEhighCmd;
```

Comment:
Please use shorter name strings.  "Emp" instead of "Empirical".  "EDep" instead of "EnergyDependent", etc.